docs/how-to/contribute-code.rst

```diff
@@ -34,16 +34,26 @@ to compile this library:
 
 This will produce ``tests/objc/librubiconharness.dylib``.
 
-You can then run the test suite:
+You can then run the full test suite:
```

Comment:
Again, GitHub doesn't let me comment on the correct location. The paragraphs above this explain how to manually call `make` to build librubiconharness.dylib. I think we should remove that part entiely - now that `make` is called automatically by our tox config, there should be no need for the user to call it manually anymore.